wg-nodeless/README.md

```diff
@@ -0,0 +1,28 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+-->
+# Nodeless Working Group
+
+Explore and ultimately propose a nodeless design that can extend a kubernetes cluster to enable pods running on other service platforms.
+
+## Meetings
+* Regular WG Meeting: [Wednesdays at 14:00 PT (Pacific Time)](https://zoom.us/j/588409652) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=14:00&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/13gt9aUokqueOYHzA94D0UXFkXSP0rJEED9TxNEXZZMM/edit?usp=sharing).
+
+## Organizers
+
+* Ria Bhatia (**[@rbitia](https://github.com/rbitia)**), Microsoft
+* Ben Corrie (**[@corrieb](https://github.com/corrieb)**), VMWare
+
+## Contact
+* [Slack](https://kubernetes.slack.com/messages/wg-nodeless)
+* [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-nodeless)
+
+<!-- BEGIN CUSTOM CONTENT -->
+The proposal and goals for the nodeless working group are outlined [here](https://docs.google.com/document/d/1Y1GEKOIB1u5P06YeQJYl9WVaUqxrq3fO8GZ7K6MUGms/edit)
```

Comment:
I think one of the big concerns is that this here shouldn't be a link to an outside document, but rather bring the content into either this README file, or another checked-in markdown file. @corrieb

Reply:
Alright, I'll delete the link and add the goals to the custom content. After I commit would we be able to merge then?

Reply:
> Alright, I'll delete the link and add the goals to the custom content. After I commit would we be able to merge then?

@rbitia I think reviewers here have requested both things you mentioned. Doing these two things will make this PR significantly different, and it will likely need another round of review. I know it is frustrating, but until the remaining changes are "nit"-sized, no one can offer a blanket assurance that the next iteration will be the one that everyone can get on board with. :(